### ISSUE TITLE
Is this being actively maintained?

### ISSUE THREAD

**Issue by TechnicallyJoe** [Azure/terraform-azurerm-naming#118]
Having a few issues here and there, most of which are solved by already existing PRs and issues.

We use this module quite extensively to ensure naming standards so i wanna see it updated.

Let me know if i can be of any assistance.

**Comment by TheIronRock95** [comment]
We also use this module quite extensively

@Nepomuceno please let me know if you need any assistance.

**Comment by Nepomuceno** [comment]
It is not a fulll time job of anyone to mantian this repo I am happy to share the maangement of it with anyone interested if anyone wants to volunteer

**Comment by TechnicallyJoe** [comment]
> It is not a fulll time job of anyone to mantian this repo I am happy to share the maangement of it with anyone interested if anyone wants to volunteer

I was actually considering forking it because we're hitting quite a few issues.

I'd love to be involved in evolving and furthering this project. Perhaps we could have a chat somewhere to discuss?

**Comment by Nepomuceno** [comment]
my email it is gabriel.bico in gmail please ping me.

**Comment by Pierca7** [comment]
Hey, if you need more maintainers I'd be happy to help too.

**Comment by davenicoll** [comment]
Volunteering to become a maintainer too!

**Comment by crpietschmann** [comment]
@TechnicallyJoe  I would be glad to help out, if volunteers are needed to maintain this project. I am a long time Microsoft MVP and HashiCorp Ambassador.

**Comment by nickel-tyler** [comment]
Turns out this project dead, and it's only maintainer is an angry loser who doesn't want help. Everyone will be better off forking their own variant, or coming up with a different solution.

**Comment by Nepomuceno** [comment]
Yes, this project is actively maintained! We recently merged PR #183 adding 51 new resources. Breaking changes are being collected for a future major version release.
